VideoLocker/src/main/java/org/edx/mobile/player/PlayerFragment.java

```diff
@@ -162,7 +163,8 @@ public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         segIO = SegmentFactory.getInstance();
         // save this fragment across activity re-creations
-        setRetainInstance(true);
+        if (!MainApplication.Q4_ASSESSMENT_FLAG)
+            setRetainInstance(true);
```

Comment:
@aleffert  I should add a TODO marker here.   we can not call setRetainInstance() directly here as playerFragment is a nested fragment now.